### DISCUSSION TITLE
Preview external links before you open them

### DISCUSSION THREAD

**Discussion by giwrgosmant** [Docile-Alligator/Infinity-For-Reddit#875] [Ideas]
It would be nice if you can preview external links (hyper links) when u click them before you open them.

**Comment by Wladefant** [comment]
@giwrgosmant Can you explain it a little bit more?
How would you like the preview to be?
Like in chrome when you mark a word?

**Comment by giwrgosmant** [comment]
> @giwrgosmant Can you explain it a little bit more?
> How would you like the preview to be?
> Like in chrome when you mark a word?

Yes, something like that.
